sql/catalyst/src/main/java/org/apache/spark/sql/catalyst/expressions/ExpressionDescription.java

```diff
@@ -112,4 +112,5 @@
     String group() default "";
     String since() default "";
     String deprecated() default "";
+    String language() default "built-in";
```

Comment:
Can you fix the documentation above? (see https://github.com/apache/spark/commit/e42dbe7cd41c3689910165458a92b75c02e70a03 FYI)

Reply:
sure